docs/conversation.md

```diff
@@ -330,6 +331,24 @@
         + `401 Unauthorized` When the participant is a guest
         + `404 Not Found` When the conversation could not be found for the participant
 
+## Set time to live for messages in a conversation
```

Comment:
Jan doesn't like "ttl" and "time to live"
Maybe we rename the database field to "chat_expiration"
and also rename the value on the APIs and the capabilities?